network_observability/metrics-alerts-dashboards.adoc

```diff
@@ -0,0 +1,17 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="metrics-dashboards-alerts"]
+= Using metrics, dashboards, and alerts
+include::_attributes/common-attributes.adoc[]
+:context: metrics-dashboards-alerts
+
+toc::[]
+
+The Network Observability Operator uses the `flowlogs-pipeline` to generate metrics out of flow logs. You can make use these metrics by setting custom alerts and viewing dashboards.
+
+include::modules/network-observability-viewing-dashboards.adoc[leveloffset=+1]
+include::modules/network-observability-metrics.adoc[leveloffset=+1]
+include::modules/network-observability-includelist-example.adoc[leveloffset=+2]
+
+[role="_additional-resources"]
+.Additional resources
+* For more information about creating alerts that you can see on the dashboard, see xref:../monitoring/managing-alerts.adoc#creating-alerting-rules-for-user-defined-projects_managing-alerts[Creating alerting rules for user-defined projects].
```

Comment:
Normally I think we want these additional resources to just be single links, without text around them.

Also, should there be an xref either here or somewhere else in the assembly for the "Creating an alert for high ingress traffic" you mention earlier?

Reply:
Yeah that is right. I will remove the surrounding text. 

The include for Creating an alert for high-ingress traffic is this: network-observability-includelist-example.adoc. The title of it has changed though to, just Creating alerts so I will adjust that reference in the module, network-observability-metrics.adoc